client/tts.py

```diff
@@ -417,6 +417,118 @@ def say(self, phrase):
         os.remove(fname)
 
 
+class BingTTS(AbstractMp3TTSEngine):
+    """
+    Uses the Bing TTS online translator
+    Appropiated from: https://github.com/wilfilho/BingTranslator
```

Comment:
That's a problem. That code is licensed under GPL, while Jasper is MIT. So we can't incorporate that code without @wilfilho's permission. What we *can* do is use the API of that module (which is probably a good idea in any case).